### ISSUE TITLE
Broken in Ember 1.13

### ISSUE THREAD

**Issue by omghax** [IvyApp/ivy-tabs#2]
See emberjs/ember.js#11170.

**Comment by jherdman** [comment]
Was this resolved?

**Comment by omghax** [comment]
This has been resolved with the 1.2.0 release, though it requires some changes to the way the components are used in the templates:

```handlebars
{{#ivy-tabs}}
  {{#ivy-tab-list}}
    {{#ivy-tab}}tab 1{{/ivy-tab}}
    {{#ivy-tab}}tab 2{{/ivy-tab}}
  {{/ivy-tab-list}}
  {{ivy-tab-panel}}panel 1{{/ivy-tab-panel}}
  {{ivy-tab-panel}}panel 2{{/ivy-tab-panel}}
{{/ivy-tabs}}
```

would become:

```handlebars
{{#ivy-tabs as |tabsContainer|}}
  {{#ivy-tab-list tabsContainer=tabsContainer as |tabList|}}
    {{#ivy-tab tabList=tabList}}tab 1{{/ivy-tab}}
    {{#ivy-tab tabList=tabList}}tab 2{{/ivy-tab}}
  {{/ivy-tab-list}}
  {{ivy-tab-panel tabsContainer=tabsContainer}}panel 1{{/ivy-tab-panel}}
  {{ivy-tab-panel tabsContainer=tabsContainer}}panel 2{{/ivy-tab-panel}}
{{/ivy-tabs}}
```

The old `parentView`-based way _should_ still work, but I noticed some of the tests were failing in 1.13, so it seemed like passing block arguments around was the better solution long-term. Eventually once I'm able to drop support for Ember versions < 2.3, I plan to make this use [contextual components](http://emberjs.com/blog/2016/01/15/ember-2-3-released.html#toc_contextual-components) instead.
